static/js/components/GroupSources.jsx

```diff
@@ -196,28 +188,57 @@ const GroupSources = ({ route }) => {
     );
   };
 
+  const renderAlias = (dataIndex) => {
+    const { id: objid, alias } = sources[dataIndex];
+
+    return (
+      <a href={`/source/${objid}`} key={`${objid}_alias`}>
+        {alias}
+      </a>
+    );
+  };
+
   // This is just passed to MUI datatables options -- not meant to be instantiated directly.
 
   const renderRA = (dataIndex) => {
     const source = sources[dataIndex];
-    return (
-      <div key={`${source.id}_ra`}>
-        {source.ra}
-        <br />
-        {ra_to_hours(source.ra)}
-      </div>
-    );
+    return <div key={`${source.id}_ra`}>{source.ra}</div>;
+  };
+
+  const renderRA_sex = (dataIndex) => {
+    const source = sources[dataIndex];
+    return <div key={`${source.id}_ra_sex`}>{ra_to_hours(source.ra)}</div>;
   };
 
   // This is just passed to MUI datatables options -- not meant to be instantiated directly.
   const renderDec = (dataIndex) => {
+    const source = sources[dataIndex];
+    return <div key={`${source.id}_dec`}>{source.dec}</div>;
+  };
+
+  const renderDec_sex = (dataIndex) => {
+    const source = sources[dataIndex];
+    return <div key={`${source.id}_dec_sex`}>{dec_to_hours(source.dec)}</div>;
+  };
+
+  const renderRedshift = (dataIndex) => {
+    const source = sources[dataIndex];
+    return <div key={`${source.id}_redshift`}>{source.redshift}</div>;
+  };
+
+  const renderClassification = (dataIndex) => {
     const source = sources[dataIndex];
     return (
-      <div key={`${source.id}_dec`}>
-        {source.dec}
-        <br />
-        {dec_to_hours(source.dec)}
-      </div>
+      <Suspense fallback={<div>Loading classifications</div>}>
+        <ShowClassification
```

Comment:
Only show the most recent classification in this table

Reply:
The ShowClassification component shows only the latest classification...

Reply:
Agreed - the ShowClassification component will display the most recent classification for each taxonomy, not just overall. 

I think the best way to do this would be to refactor the ShowClassification component slightly to have an prop for toggling showing just the most recent classification vs for each taxonomy. It seems like something we could want to reuse later. Honestly, maybe just use the `shortened` flag you already put in and say, if shortened, only display the most recent classification.

Reply:
I just wanted to leave the behavior of ShowClassifications as it was for other people using it. I agree that we can make this component more versatile. I will have to make sure it doesn't appear in too many other places where you'd suddenly get all classifications instead of one.

Reply:
On the group sources page people will only be interested in the classification in the context of their group. Please refactor this to show only the latest classification from the group that is being shown